content/en/docs/tasks/administer-cluster/verify-signed-images.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Verify Signed Container Images
+
+reviewers:
+  - TBD
+
+content_type: task
+min-kubernetes-server-version: v1.24
+---
+
+<!-- overview -->
+
+{{< feature-state state="alpha" for_k8s_version="v1.24" >}}
+## Pre-requisites
+- [cosign](https://docs.sigstore.dev/cosign/installation/)
+
+## Verifying image signatures
+For a complete list of images that are signed please refer to [releases](../../../../releases/download) page.
+
+Let's pick one image from this list and verify its signature using `cosign verify` command.
+
+```shell
+COSIGN_EXPERIMENTAL=1 cosign verify k8s.gcr.io/kube-apiserver-amd64:v1.24.0
+```
+### All control plane images
```

Comment:
Let's add another section to mention cosigned: https://github.com/sigstore/cosign/tree/main/config

Reply:
added